pkg/modules/dependencies.go

```diff
@@ -6,14 +6,16 @@ const (
 
 	Core                  string = "core"
 	GrafanaAPIServer      string = "grafana-apiserver"
+	Ring                  string = "ring"
```

Comment:
There can be many different rings in the future, shall we call it "StorageRing" or something more suitable?